modules/programs/mako.nix

```diff
@@ -0,0 +1,264 @@
+        description = ''
+           Show notifications at the specified position on the output. Supported values are top-right, top-center, top-left, bottom-right, bottom-center, bottom-left, and center.
+         '';
+      };
+
+      font = mkOption {
+        default = "monospace 10";
+        type = types.nullOr types.str;
+        description = ''
+	   Set font to font, in Pango format.
+        '';
+      };
+
+      backgroundColor = mkOption {
+        default = "#285577FF";
+        type = types.nullOr types.str;
+        description = ''
+	   Set background color to color. See COLORS for more information.
+        '';
+      };
+
+      textColor = mkOption {
+        default = "#FFFFFFFF";
+        type = types.nullOr types.str;
+        description = ''
+	   Set text color to color. See COLORS for more information.
+        '';
+      };
+
+      width = mkOption {
+        default = 300;
+        type = types.nullOr types.int;
+        description = ''
+	   Set width of notification popups.
+        '';
+      };
+
+      height = mkOption {
+        default = 100;
+        type = types.nullOr types.int;
+        description = ''
+           Set maximum height of notification popups. Notifications whose text takes up less space are shrunk to fit.
+        '';
+      };
+
+      margin = mkOption {
+        default = 10;
+        type = types.nullOr types.int;
+        description = ''
+           Set margin of each edge to the size specified by directional.
+	   See DIRECTIONAL VALUES for more information.
+        '';
+      };
+
+      padding = mkOption {
+        default = 5;
+        type = types.nullOr types.int;
+        description = ''
+	   Set padding on each side to the size specified by directional.
+           See DIRECTIONAL VALUES for more information.
+        '';
+      };
+
+      borderSize = mkOption {
+        default = 1;
+        type = types.nullOr types.int;
+        description = ''
+           Set popup border size to px pixels.
+        '';
+      };
+
+      borderColor = mkOption {
+        default = "#4C7899FF";
+        type = types.nullOr types.str;
+        description = ''
+           Set popup border color to color. See COLORS for more information.
+        '';
+      };
+
+      borderRadius = mkOption {
+        default = 0;
+        type = types.nullOr types.int;
+        description = ''
+	   Set popup corner radius to px pixels.
+        '';
+      };
+
+      progressColor = mkOption {
+        default = "over #5588AAFF";
+        type = types.nullOr types.str;
+        description = ''
+           Set popup progress indicator color to color. See COLOR for more information. To draw the progress indicator on top of the background color, use the over attribute. To replace the background color, use the source attribute (this can be useful when the notification is semi-transparent).
```

Comment:
Reflow this please. Remove the `COLOR` reference if we don't bundle this anywhere.